File: java/arcade/MakeArrayConsecutive2.java
import java.util.Arrays;

public class MakeArrayConsecutive2 {

	int makeArrayConsecutive2(int[] statues) {
	    Arrays.sort(statues);
		int min = statues[0];
		int max = statues[statues.length - 1];
		int count = 0;
		boolean b = false;
		for(int i = min; i < max; i++){
		    for(int j : statues) {
		        if(j == i) {
		            b = !b;
		            break;
		        }
		    }
		    count += (!b) ? 1 : 0;
		    b = false;
		}
		return count;
	}
}

File: Arcade/TheCore/Kotlin/makeArrayConsecutive2.kt
fun makeArrayConsecutive2(statues: MutableList<Int>): Int {
    val max: Int = statues.maxOf { it }
    val min: Int = statues.minOf { it }
    var additionalStatues = 0;
    
    (min..max).forEach({ statue ->
        if (!(statue in statues))
            additionalStatues++
    });
    
    return additionalStatues;
}